secrets.yaml

```diff
@@ -1,11 +1,34 @@
-build02-root: ENC[AES256_GCM,data:cNqf/Gw+d6mi5buPZu4+,iv:afT+IPf1HY98QQeBEVEfelEpcxTZHsAmDttEdyfvkWA=,tag:MUjl56D52rsHfpqt4tGcnw==,type:str]
-build02-admin: ENC[AES256_GCM,data:rTVQQnj9yBaiUnib4GAYBsoDZjs8PJ0kb6GLr3iOhhW92s26veybw3ZsdGSEJPdsE0xD9NtaN8DHfMP/bgA1gUX9y2+qcmRfkbT67LFo9fkw4AKSkC5fABtq1EeHQfE3zLMebVO2g9xnQ3w7Y/HFX8rUlYDA7GhYrTgL7FwXNE+/J6rckv9aW0GxKjvFAhqkCbXxDGHebxpIBDbgF1JldmHDFQ==,iv:YXERjZTyNGAlK/9C4i2L/P9p+P9cz46/yk/fda3ryu8=,tag:MuDFM6zg+LSoPkkmK3UFgA==,type:str]
-build04-root: ENC[AES256_GCM,data:Qy3xWgN6HCYRxAd/WArd,iv:2OT9i2YFHqiRx8cyPZ6oW/rxWNgqXdzD+Kk4dP1HbrY=,tag:JQ76gLcCau+ozMOZPcifeg==,type:str]
-nix-community-alert-bot: ENC[AES256_GCM,data:Yg69dnfFGXtMTLeyXnMjeIc8+fBmeA4lEYNwSh/ahHdVd/um8oe+CSBDhIqio2sB+GrrpoNAwMf+6rC5fK7q71eAoKcrzjz2nneJdYY=,iv:fbF8t1ySI4qJ+uj33k2N9C/x7bCFdR5OBI/CGhz3zC8=,tag:xcY0Zxtb/w9vRpA56ro8Hw==,type:str]
-nix-infra-bot: ENC[AES256_GCM,data:+YRdg8wzRtAUF5GbWvO++qgcyi0st/nMu5PS/IgcBpFj1Zsazta7URuLkSiDFpZDzQYt3bgRzv2dXjPS23hZFoLqgaYysq1E8RF+lvQYHbkMXfR8d8DeFIOtkucW1EIdYMnzEgvfonbF1j3fiy7aDvzDjMK0CJrHzqnDpiYb7W0rMCTyuOyrm0RVw91GN6VTZOrR3dOykJoO4di0qLMlQ2hdYNQi3XQBntvB58gjr3AmtUhuSfw9F4UXdqAPLXi1FnT5rxpBEA09N9Y=,iv:LFqq3WvxlOaz2hQpbcjMDXWWdGqJRjLSQLLZdypHK6I=,tag:PG5sECQOZ6AKW8eGbQSA8Q==,type:str]
-cloudflare.com: ENC[AES256_GCM,data:HtBSlzYtzeV4qrBlNdQXNQGRwEt/4dKt6iH4+ryOHib5HDJxJhG6zYneItFxQ10/fuJlXqLizfladsUzADvvEwCp4KJ06HaZn1do56J3lelDz+ROOuKzvTncGt/Fu4rjwT2SSHXmV+XTy8B3iBXDoX/d/u87J+j5y+BLY2XDLcq7YMDizyV/qg4kboKp9r5lravvNx2ZOj9T1TnKkJBJ32lH7Iz7SHo4qYOc/IBGXAS+e2qGVPCLWuygKg3mxTJ2Y/y6ba6JuC97fEo=,iv:dOACToYa8S44ANNIsHktD/lAG+VR/+mMqcdF8n77yaA=,tag:eqtUmKOGb/hFLIRl0FXNeA==,type:str]
-hetzner.com: ENC[AES256_GCM,data:rcYjYibR7lrdr8JTJ/ljS913kTjsC2J8HEcORA15x52l7+yYFS6FCsrWanFQaM8yc2w7NDp6+ND8PhQh5PLzuIJZyWHcosetJ9/Ow3SJjqNMnH2GogD/23iS3zwg2gKvnyMp/7gDIUkzsqHVysx3GIWjzq7Hdyq8Ls9/GIPqU8VYccR+nQ==,iv:kXygbJHHIJUkz/NAuAPBxqGAepLO/o3WVYz6YEwmrio=,tag:SKOCB1T8Cv5qkH856UAHvw==,type:str]
-darwin02-m1: ENC[AES256_GCM,data:0WKa4cWpEqWvZrEn,iv:ig+gD7ucGVh2WKVzmuxMiM9A2RJVM0E5avRzzc2Zqjk=,tag:6y0rEvjui0spqeE9uP+L7w==,type:str]
+build02:
+    root: ENC[AES256_GCM,data:XCyC99IsbFlMrL0iNazU,iv:CF+QthpBj4fJtUDLh4T+2ChDgfHKO1qZsO68usASIxA=,tag:83CGuJKxjsEu6DdkcN4QOQ==,type:str]
```

Comment:
Is this password still used? I don't see password hashes being set, but maybe we should.

Reply:
Don't think it's used, also current ssh settings don't allow root with a password.

Guess we can drop `root` for `build02` and `build04`?

> I don't see password hashes being set, but maybe we should.

Not sure why we'd want this?

Reply:
One can get a KVM console from hetzner for troubleshooting. There it might come in handy.

Reply:
I don't really see that we need it but I won't object if you want to set them.

Would you like to keep these current passwords or will you generate new passwords?

Reply:
You can drop this password for now. If I add new ones, I can do another pull request.